src/mediapipe_internal/mediapipegraphconfig.hpp

```diff
@@ -53,20 +53,45 @@ class MediapipeGraphConfig {
          */
     std::string rootDirectoryPath;
 
+    /**
+     * @brief Json config path
+     */
+    std::string modelsConfigPath;
+
+    void setPath(std::string& path, const std::string& givenPath) {
+        path = givenPath;
+
+        if (!FileSystem::isLocalFilesystem(givenPath)) {
```

Comment:
Check this in ovms repo:
grep -r -R -A8 -B3 isLocalFilesystem

we already have similar function bodies in mediapipegraphconfig.hpp, customloaderconfig.hpp, modelmanager.cpp, modelconfig.hpp. Maybe we can have utility function? @rasapala